docs/reference/cat/tasks.asciidoc

```diff
@@ -13,8 +13,7 @@ console. They are _not_ intended for use by applications. For application
 consumption, use the <<tasks,task management API>>.
 ====
 
-Returns information about tasks currently executing in the cluster,
-similar to the <<tasks,task management>> API.
+Returns information about tasks currently executing on <<modules-node,nodes>> in the <<modules-cluster,cluster>>.
```

Comment:
I don't think these links add value. The thing the person needs information about is what a task is - maybe an example would be more useful here.

Reply:
So
1. I feel strongly to highlight that CAT Tasks are not "tasks executing in the cluster" on the CLUSTER level vs NODE level which keeps confusing users based on what this currently says. 
2. I feel much less strongly, but my goal is to link each API to a parent object reference &, while I agree there is no explaining Node Task docs which y'all requested to back burner [in Slack Sep25](https://elastic.slack.com/archives/CJE3WCFEH/p1727280598509239), at least the nodes link is _as close_ as our docs currently get. 

Thoughts?

Reply:
>my goal is to link each API to a parent object reference

I don't think we should add links for links' sake. I'd remove the link and just clarify the definition.